app/src/main/java/com/example/cocktaildictionary/actions/HomeActivityAction.kt

```diff
@@ -0,0 +1,13 @@
+package com.example.cocktaildictionary.actions
+
+import com.example.cocktaildictionary.network.CocktailList
+
+/*
+*These are all of the possible actions that can be triggered from the Loading screen
+*/
+
+sealed class HomeActivityAction: Action {
+    object LoadingStarted : HomeActivityAction()
+    data class LoadingSuccess(val newData: CocktailList?) : HomeActivityAction()
```

Comment:
These params are not optional right? Similar to `HomeActivityStates`

Reply:
No, these params are not optional. They are necessary because when the LoadingSuccess Action is called and returns the HomeActivityStates.DataLoadSuccessful(cocktails: CocktailList) the newData which was passed as a parameter for LoadingSuccess Action will be extracted and passed as the parameter for HomeActivityStates.DataLoadSuccessful(cocktails: CocktailList)

Reply:
Cool, then remove `?`

Reply:
Ohkay. I removed it.